core/controllers/editor.py

```
@@ -145,10 +154,10 @@ def put(self, exploration_id):
             self.user, exploration_rights)
 
         try:
-            if can_edit:
+            if can_edit and changes_are_mergeable:
```

If changes are not mergeable we should raise an error instead of sending 200 responses, right? Also, are we going to show a modal in UI if the `changes_are_mergeable` is False?

---

No, I don't think so that we should raise an error when the changes are not mergeable because we are already showing the UI modal directly.
Yea we show a modal in UI if the ```changes_are_mergeable``` is False.